Feat/appcircle cli doc (#23)
appcircleio/appcircle-cli: docs/config/get.md

# `appcircle config get`

> Get Print the value of a Appcircle CLI currently active configuration property.

Reviewer: ```suggestion
Print the value of the currently active configuration property in the Appcircle CLI.
```
